userspace/libscap/plugin_info.h

```diff
@@ -359,7 +357,7 @@ typedef struct
 	//   by the plugin and must not be deallocated or modified until the next
 	//   extract_fields() call.
 	//
-	// Return value: An ss_plugin_rc with values SS_PLUGIN_SUCCESS or SS_PLUGIN_FAILURE.
+	// Return value: A ss_plugin_rc with values SS_PLUGIN_SUCCESS or SS_PLUGIN_FAILURE.
```

Comment:
This reads to me as "an ess-ess-plugin-rc" so I'd leave the "An" (elsewhere as well). But I'm not going to fight over it so I'll ✅ if you decide "A" is what you prefer.

Reply:
I followed the crowd... more "a" than "an" :shrug: